### ISSUE TITLE
Can you use Environment Variable for API_KEY and API_SECRET?

### ISSUE THREAD

**Issue by glaksmono** [veritrans/veritrans-ruby#23]
https://github.com/veritrans/veritrans-ruby#edit-api-keys-in-configveritransyml

Not a good practice if we have to hardcode this on the .yml file

Can we do the following:

```
staging:
  client_key: <%= ENV['VERITRANS_CLIENT_KEY'] %>
  server_key: <%= ENV['VERITRANS_SERVER_KEY'] %>
  api_host: https://api.sandbox.veritrans.co.id

production:
  client_key: <%= ENV['VERITRANS_CLIENT_KEY'] %>
  server_key: <%= ENV['VERITRANS_SERVER_KEY'] %>
  api_host: https://api.veritrans.co.id
```

Or we need to hard-code it for now?

**Comment by yocki-s** [comment]
Hi @glaksmono 

Yes you can do that. The point for us is, please make sure you are using right key and endpoint.
ex: If you use sandbox endpoint, please make sure you are using sandbox key.

**Comment by bprayudha** [comment]
@yocki-s I believe this [commit](https://github.com/veritrans/veritrans-ruby/commit/62b1ec68744d9cf156ee4f0743a825d0260a371c) can fix this issue but it's not released yet on [rubygems.org](https://rubygems.org/gems/veritrans).

**Comment by yocki-s** [comment]
hi @bprayudha 

Thanks for your information. i will check this

**Comment by Paxa** [comment]
Hi @glaksmono 
I just released new version `2.1.1` with that feature. Please try
